chat-plugins/info.js

```diff
@@ -460,6 +288,13 @@ var commands = exports.commands = {
 		}
 		this.sendReply(buffer);
 	},
+	datahelp:
+		"/data [pokemon/item/move/ability] - Get details on this pokemon/item/move/ability/nature.\n" +
+		"!data [pokemon/item/move/ability] - Show everyone these details. Requires: + % @ & ~",
+	detailshelp:
+		"/details [pokemon] - Get additional details on this pokemon/item/move/ability/nature.\n" +
+		"!details [pokemon] - Show everyone these details. Requires: + % @ & ~",
+	dthelp: function () { this.parse('/help details'); },
```

Comment:
No, I mean, make `data` and `details` into separate functions instead of aliases of each other.